src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -502,20 +492,19 @@ namespace ExhaustAirSystemManager {
                 ip->markObjectAsUsed(cCurrentModuleObject, instance.key());
 
                 std::string availSchName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "availability_schedule_name");
```

Comment:
📌 How many times does the string value get copied in `getAlphaFieldValue`?

Reply:
It seems at least the last parameter of this function can be passed as `string_view`:
```
std::string InputProcessor::getAlphaFieldValue(json const &ep_object, json const &schema_obj_props, std::string const &fieldName)
{
    // Return the value of fieldName in ep_object as a string.
    // If the field is not present in ep_object then return its default if there is one, or return an empty string
    auto const &schema_field_obj = schema_obj_props[fieldName];
```

The local string `value` needs to be a writable one, so probably nothing could do on it: 
```
    std::string value;
    auto it = ep_object.find(fieldName);
    if (it != ep_object.end()) {
        auto const &field_value = it.value();
        if (field_value.is_string()) {
            auto valuePair = getObjectItemValue(field_value.get<std::string>(), schema_field_obj);
            value = valuePair.first;
            isDefaulted = valuePair.second;
        } else {
            assert(false); // String value requested but field type is numeric
        }
    } else {
        isDefaulted = findDefault(value, schema_field_obj);
        if (!isDefaulted) {
            value = "";
```